cli/fathom_web/test/test_vectorize.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from click.testing import CliRunner
+
+from ..commands.vectorize import main as vectorize_main
+
+
+def test_end_to_end(tmp_path):
+    test_dir = os.path.dirname(os.path.abspath(__file__))
+
+    runner = CliRunner()
+    result = runner.invoke(
+        vectorize_main,
+        [
+            f'{test_dir}/resources/vectorize/vectorize_ruleset.js',
+            'secret',
+            f'{test_dir}/resources/vectorize/',
+            os.environ['FATHOM_FOX'],
+            os.environ['FATHOM_TRAINEES'],
```

Comment:
Unlike all our other `fathom-*` tests, this test [does not pass locally](https://paste.mozilla.org/EMVQBpw0) for me:
```
`E           KeyError: 'FATHOM_FOX'
../../.pyenv/versions/3.7.0/lib/python3.7/os.py:678: KeyError`
```
Though we use the environment variables in our CI, we don't define them when the test is run locally. I'm not sure the best way to solve this, as we don't know where the developer may have `fathom-fox` and `fathom-trainees` installed locally. What do you think? Also @erikrose ?

Reply:
I'm interested in both of your thoughts on needing to set this environment variable if you want to run the tests locally. I had the additional thought that `fathom-vectorize` could look for the environment variable and use them if they are present so the user doesn't have to specify the paths every time they use the tool.

Reply:
One option is, as part of the test, do what the CI does -- i.e. clone both repos and set them to the two environment variables -- just clean up at the end of the test. This may further help in situations where the user's local copies of either or both repos is not clean, and could cause the test to fail. I'm not sure then what the `.travis.yml` file would look like, however.

Reply:
We should agree on something here before merging this.

Reply:
If the checkout is done as part of `py_test` in the `Makefile` that could work. I'll mess around with that.

Reply:
Okay, the real plan is to do this in the setup for the test in Python as was suggested by both @biancadanforth and @erikrose. I don't think I'll get this working before I go on leave though.